charts/hivemq-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ keywords:
   - messaging
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.11.6
+version: 0.12.0
```

Comment:
Is the version numbering change coordinated with Simon?

Reply:
hm, i'm not sure what you mean by this? so far the versioning "convention" (which isn't documented anywhere AFAIK though) was to bump the minor version when templates change. a quick diff shows though that templates have already changed within 0.11.x so 🤷‍♂️  not sure what to do

i think we can stay within 0.11.x though because no template is changing within this version IMO.

Reply:
Regarding semantic versioning https://semver.org the minor is done when there are not breaking changes but functionality is added, this change is for me on the greyline, yes is only a change on the templates but add functionally, indirectly. Now the questions is then, change on the values does not add functionality but templates do? That seems strange, in either case seems highly subjective.

Reply:
FYI: https://github.com/hivemq/helm-charts/pull/83#discussion_r976096649